assets/quick-buy.css

```diff
@@ -0,0 +1,197 @@
+.quick-buy-modal {
+  box-sizing: border-box;
+  opacity: 0;
+  position: fixed;
+  visibility: hidden;
+  z-index: -1;
+  margin: 0 auto;
+  top: 0;
+  left: 0;
+  overflow: auto;
+  width: 100%;
+  background: rgba(var(--color-foreground), 0.2);
+  height: 100%;
+}
+
+.quick-buy-modal[open] {
+  opacity: 1;
+  visibility: visible;
+  z-index: 101;
+}
+
+.quick-buy-modal__content {
+  border-radius: var(--popup-corner-radius);
+  background-color: rgb(var(--color-background));
+  overflow: auto;
+  margin: 0 auto;
+  left: 50%;
+  transform: translateX(-50%);
+  margin-top: 3.2rem;
```

Comment:
```suggestion
  margin:3.2rem auto 0;
  left: 50%;
  transform: translateX(-50%);
```

Also on mobile it seems to cut the top border when there is one: ![](https://screenshot.click/03-23-trzkl-vn2uo.png)

Reply:
good catch! I marked this spacing to be 40px on one of my previous comments above.